webrtc-stats.html

```diff
@@ -1529,6 +1530,30 @@ <h2>
                   6.3.2.
                 </p>
               </dd>
+              <dt>
+                <dfn>totalPlayoutDelay</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  It is the sum of the time, in seconds, each <a>audio sample</a> or video frame takes from
+                  the time the first RTP packet is received (reception timestamp) and to the time 
+                  the corresponding sample or frame is delivered to the MediaStreamTrack for playout (playout timestamp). 
+                  Given the complexities involved, the time of arrival or the reception timestamp is measured 
+                  as close to the network layer as possible and the playout timestamp is measured as close to
+                  the rendering as possible.
+                  In the case of audio, several samples are received in the same RTP packet, all samples
+                  will share the same reception timestamp and different playout timestamps. 
+                  In the case of video, the frame are received over several RTP packets, in this
+                  case the earliest timestamp containing the frame is counted as the reception timestamp,
+                  and the reception timestamp corresponds to when the complete frame is played out.
```

Comment:
Change "complete frame is played out" to "complete frame is ready to be played out"

Reply:
reception timestamp should say playout timestamp here